py_gtfs_rt_ingestion/ingest.py

```diff
@@ -164,8 +173,7 @@ def lambda_handler(event: LambdaDict, context: LambdaContext) -> None:
     logging.info("Context:%s", context)
 
     try:
-        files = event["files"]
-        main(files)
+        main(event)
```

Comment:
just process the event dict directly. i think its still a good idea to wrap `main` in `lambda_handler` though.